jmui/setup.py

```diff
@@ -0,0 +1,16 @@
+from setuptools import setup
+import os
+
+setup(name='joinmarketui',
+      version='0.9.0dev',
+      description='Joinmarket client library for Bitcoin coinjoins',
+      url='http://github.com/Joinmarket-Org/joinmarket-clientserver/jmfg',
+      author='',
+      author_email='',
+      license='GPL',
+      packages=['jmui'],
+      install_requires=['PyQt5!=5.15.0,!=5.15.1,!=5.15.2,!=6.0'],
```

Comment:
Copied from `requirements/gui.txt`, but I'm not quite certain whether this is the right way to declare the dependency of PyQt5.

Reply:
See #913 for context, 5.15.0, 5.15.1, 5.15.2 and 6.0 had a bug for some users which is now fixed with newer versions.